src/components/action-bar/action-bar-item/action-bar-item.tsx

```diff
@@ -149,24 +156,24 @@ export class ActionBarButton {
             return;
         }
 
-        if (this.item.text) {
-            return (
-                <limel-tooltip
-                    elementId={this.tooltipId}
-                    label={this.item.text}
-                    helperLabel={this.item.commandText}
-                />
-            );
-        }
+        return (
+            <limel-tooltip
+                elementId={this.tooltipId}
+                label={this.getTooltipLabel(this.item)}
+                helperLabel={this.item.commandText}
+            />
+        );
+    }
 
-        if (this.item.commandText) {
-            return (
-                <limel-tooltip
-                    elementId={this.tooltipId}
-                    label={this.item.commandText}
-                />
-            );
+    private getTooltipLabel(item: ActionBarItem): string {
+        const iconTitle = getIconTitle(item.icon);
+        const tooltipLabel = item.text;
```

Comment:
This is causing the tests on dev-feature to fail
https://github.com/Lundalogik/solution-dev-feature/commit/f4d6f577ea49228ff477532a3a4d8fa8be6f2a28/checks
`item.text` seems to be `undefined`

Reply:
It seems to get its `undefined` from here
https://github.com/Lundalogik/lime-elements/blob/main/src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx#L283

Reply:
There is a typing error here 
https://github.com/Lundalogik/lime-elements/blob/main/src/global/translations.ts#L22
It should probably be changed to 
```diff
export class Translations {
-    public get(key, language = 'en') {
+    public get(key, language = 'en'): string | undefined {
        return allTranslations[language][key];
    }
}
```

Reply:
Type information has also been lost here, which is not helping
https://github.com/Lundalogik/lime-elements/blob/main/src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx#L276-L277

Reply:
I hope this PR makes things better: https://github.com/Lundalogik/lime-elements/pull/3487